backend/open_api_spec.yml

```diff
@@ -0,0 +1,729 @@
+        Determines the visibility of the collection, that will either make the collection visible on the public
+        sites or only viewable through obfuscated URLs.
+      type: string
+      enum: [PUBLIC, PRIVATE]
+    distribution:
+      description: >-
+        Cellxgene runs a heuristic to detect the approximate distribution of the data in X so that it can accurately
+        calculate statistical properties of the data. This field enables the curator to override this heuristic
+        and specify the data distribution explicitly.
+      type: string
+      enum: [COUNT, NORMAL]
+    is_primary_data:
+      description: >-
+        Describes whether cellular observations for this dataset are all canonical (PRIMARY),
+        all non-canonical (SECONDARY), or contain a mixture (BOTH).
+      type: string
+      enum: [PRIMARY, SECONDARY, BOTH]
+    links:
+      type: array
+      items:
+        type: object
+        additionalProperties: false
+        required:
+          - link_url
+          - link_type
+        properties:
+          link_name:
+            type: string
+          link_url:
+            type: string
+          link_type:
+            type: string
+            enum: [PROTOCOL, RAW_DATA, DOI, LAB_WEBSITE, OTHER, DATA_SOURCE]
+    dataset:
+      type: object
+      required:
+        - id
+        - curator_tag
+        - name
+        - revision
+        - collection_id
+        - collection_visibility
+        - revised_at
+        - is_primary_data
+        - X_normalization
+        - dataset_assets
+        - tombstone
+        - assay
+        - organism
+        - tissue
+      properties:
+        id:
+          $ref: "#/components/schemas/dataset_uuid"
+        curator_tag:
+          $ref: "#/components/schemas/curator_tag"
+        name:
+          type: string
+        revision:
+          type: integer
+        collection_id:
+          $ref: "#/components/schemas/collection_uuid"
+        collection_visibility:
+          $ref: "#/components/schemas/visibility"
+        revised_at:
+          type: number
+        is_primary_data:
+          $ref: "#/components/schemas/is_primary_data"
+        X_normalization:
+          type: string
+        dataset_assets:
+          type: array
+          items:
+            $ref: "#/components/schemas/dataset_asset"
+        tombstone:
+          type: boolean
+        assay:
+          $ref: "#/components/schemas/ontology_element_array"
+        organism:
+          $ref: "#/components/schemas/ontology_element_array"
+        tissue:
+          $ref: "#/components/schemas/ontology_element_array"
+        disease:
+          $ref: "#/components/schemas/ontology_element_array"
+        sex:
+          $ref: "#/components/schemas/ontology_element_array"
+        ethnicity:
+          $ref: "#/components/schemas/ontology_element_array"
+        development_stage:
+          $ref: "#/components/schemas/ontology_element_array"
+        dataset_deployments:
```

Comment:
I've never understood why `dataset_deployments` is an array since it simply contains the url for the CXG permanent link:

```
        "dataset_deployments": [
          {
            "url": "https://cellxgene.cziscience.com/e/aad97cb5-f375-45ef-ae9d-178e7f5d5180.cxg/"
          }
```

Reply:
I would recommend the following changes:

1. This does not need to be an array.
2. The name is not self-documenting. What is a "dataset deployment" to a curator. @jahilton @ambrosejcarr - any suggestions for a rename related to visualization, explorer, cxg, etc?

Reply:
`visualization_url` or `explorer_url`